contracts/JBXBuybackDelegate.sol

```diff
@@ -332,6 +341,27 @@ contract JBXBuybackDelegate is JBOwnable, ERC165, IJBFundingCycleDataSource, IJB
         emit JBXBuybackDelegate_TwapDeltaChanged(_oldDelta, _newDelta);
     }
 
+    /**
+     * @notice Sweep the eth left-over in this contract
+     */
+     function sweep(address _beneficiary) external {
+        // The beneficiary ETH balance in this contract leftover
+        uint256 _balance = sweepBalanceOf[_beneficiary];
+
+        // If no balance, don't do anything
+        if (_balance == 0) return;
+
+        // Reset beneficiary balance
+        sweepBalanceOf[_beneficiary] = 0;
+
+        // Keep the contract balance up to date
+        sweepBalance = address(this).balance;
```

Comment:
we should reduce this 
so I feel we need this instead
`sweepBalance = address(this).balance - _balance`